mongomock/collection.py

```diff
@@ -809,7 +809,7 @@ def _copy_only_fields(self, doc, fields, container):
                 else:
                     doc_copy = self._copy_field(doc, container)
             # if 1 was passed in as the field values, include those fields
```

Comment:
Might want to change this to "if a truthy value", to reflect the change.

Reply:
Done. Thanks.